app/views/works/show.html.erb

```diff
@@ -240,9 +246,11 @@
   <% end %>
 </div>
 
+<%= javascript_include_tag 'show_work_utils' %>
```

Comment:
If this is part of the manifest, is it really needed here too?

Reply:
I saw that we did that for the other files and I just did the same (see https://github.com/pulibrary/pdc_describe/blob/main/app/views/works/edit.html.erb#L36 and https://github.com/pulibrary/pdc_describe/blob/main/app/assets/config/manifest.js#L3)